### PR TITLE
Taskbar button to be always visible

### PR DESCRIPTION
**Pull request by EstebanDalelR**

### CODE REVIEW

**Review by EstebanDalelR — COMMENTED** [review]
Interesting, the `.js` and `.map.js` files were on the gitignore

File: images/gitlens-activitybar.svg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24">
```
Comment (EstebanDalelR) [inline images/gitlens-activitybar.svg:1]: Using gitlens to test.
It works

File: images/wmbw24.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...iigAooooAKKKKACiiigAooooAKKKKACiiigAooooATtRR2ooAWiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKAE7UUdqKAFooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigBO1FHaigBaKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooATtRRRQB//2Q=="/></g></g></svg>
```
Comment (EstebanDalelR) [inline images/wmbw24.svg:1]: This one seems to be a blob. Gotta get it fixed.

File: package.json [hunk 3]
```diff
@@ -1,64 +1,100 @@
...
-		"eslint": "^7.21.0",
-		"typescript": "^4.4.3",
-		"vsce": "^2.6.7"
-	},
-	"dependencies": {
-		"@octokit/core": "^3.5.1",
-		"@octokit/rest": "^18.12.0",
-		"@vscode/test-electron": "^2.1.2"
-	}
+        "command": "watermelon.signin",
```
Comment (EstebanDalelR) [inline package.json:40]: gotta split it to have the navbar button

**Comment by baristaGeek** [comment]
I would remove the Git Lens file and merge* with the current icon while we get a new SVG, but the current one is good enough. Good job. 

* Is the web panel gonna get moved here? Or should that be treated as an Epic?

**Review by baristaGeek — APPROVED** [review]
I can see Tulia now. Approved.

**Pull request merged**